dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/internal/ApplicationProcessor.java

```diff
@@ -345,6 +353,11 @@ private OpenAPI generateModel(OpenApiConfig config, Container appContainer, WebM
             }
 
         } finally {
+            // Clean up smallrye-open-api thread locals
+            SchemaRegistry.remove();
+            JavaSecurityProcessor.remove();
+            CurrentScannerInfo.remove();
```

Comment:
I'm sure that these calls are needed at some point but they look strange... `remove` is an odd method name and these classes are not directly used in the try block above?

Reply:
I will add a better comment here